recipes/refnx/run_test.py

```diff
@@ -0,0 +1,9 @@
+import sys
+
+import refnx
+import refnx.analysis
+import refnx.util
+import refnx.dataset
+import refnx.reduce
```

Comment:
I'm going to remove all these imports, only `import refnx` is needed for the test to run.
I do have a question here though, from conda build documentation one is supposed to list imports present in `run_test.py` in the yaml file. However, it's not explained why. I think having the answer to this question in FAQ would be helpful.